lib/dogapi/common.rb

```diff
@@ -129,6 +129,9 @@ def request(method, url, extra_params, body, send_json, with_app_key=true)
           end
 
           resp = conn.request(req)
+          if resp.code.to_i / 100 == 3
+            resp = handle_redirect(conn, req, resp, 0)
```

Comment:
```suggestion
            resp = handle_redirect(conn, req, resp)
```
or adding the `retries` param to `def request(method, url, extra_params, body, send_json, with_app_key=true, retries=10)`